assets/templates/header.html

```diff
@@ -46,7 +46,7 @@
     <li class="nav-item"><a href="forgot-password.html" class="nav-link">Forgot</a></li>
   </ul><!-- nav -->
 </section><!-- section-module -->
-    <header class="header-main header-is-active">
+    <header class="header-main header-open">
```

Comment:
Does it requires changes in webapp?

Reply:
Yeah, and I'lll push a PR on the webapp with the changes as well.